pynws/simple_nws.py

```diff
@@ -245,9 +260,7 @@ def observation(self):
             else:
                 data[obs] = obs_item
 
-            if data[obs] is None and (
-                met is not None and self._metar_obs[0] is not None
-            ):
+            if not data[obs] and met and self._metar_obs and self._metar_obs[0]:
```

Comment:
Isn't `data[obs]` here a numberic value?  If so then this change to checking Falsey will be a bug (although I agree with cleaning it up).

Reply:
Fixed